src/js/views/patients/patient/sidebar/sidebar_views.js

```diff
@@ -6,6 +6,7 @@ import hbs from 'handlebars-inline-precompile';
 import intl from 'js/i18n';
 
 import 'sass/modules/widgets.scss';
+import 'sass/domain/patient-sidebar.scss';
```

Comment:
I think the `domain` imports should go right above the local scss imports to insure they occur after all general modules.  FWIW I don't think this is documented adequately in our readme.

Reply:
Updated in the latest commit.